### PR TITLE
docs: a little bit of README

### PR DESCRIPTION
**Pull request by alexander-fenster**
Adding some very basic `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,88 @@
...
+
+### Run the generator to generate TypeScript libraries
+
+The easiest way to get started is to use our Docker image:
+
+```sh
+$ mkdir showcase-typescript
+$ docker run --rm \
+  --mount type=bind,source=`pwd`/google/showcase/v1beta1,destination=/in/google/showcase/v1beta1,readonly \
+  --mount type=bind,source=`pwd`/showcase-typescript,destination=/out \
```
Comment (xiaozhenliu-gg5) [inline README.md]: should we give users the place to download the docker image? Or we will add it later, thanks!

File: README.md [hunk 2]
```diff
@@ -1,10 +1,88 @@
...
+### Write your JavaScript or TypeScript code that uses the server!
+
+Assuming you have Node.js installed and `node` binary in your path:
+
+```sh
+$ node
+Welcome to Node.js v12.1.0.
+Type ".help" for more information.
+> const showcase = require('.')  // assuming you're in showcase-typescript
+undefined
```
Comment (xiaozhenliu-gg5) [inline README.md]: is this `undefined` intentional?

Comment (alexander-fenster) [reply README.md]: Yes, I'm just showing verbatim Node.js output.

File: README.md (hunk 1 above)
Comment (alexander-fenster) [reply README.md]: That's exactly the command that should be used, according to [AIP 4290](https://aip.dev/4290). There is no need to download Docker image, `docker run` will pull the image from gcr.io.

**Pull request merged**